Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java

```diff
@@ -1416,7 +1415,7 @@ private void checkMissingAbilities(Card card, MtgJsonCard ref) {
                     .map(Ability::getAllEffects)
                     .flatMap(Collection::stream)
                     .anyMatch(effect -> effect.getClass().isAssignableFrom(objectClass));
-            mustCheck = false; // TODO: enable and fix all problems with effect and ability hints
+            mustCheck = true; // TODO: enable and fix all problems with effect and ability hints
```

Comment:
Should I be removing this?

Reply:
Yes. If all checks passed then you can remove TODO and enable check part forever (e.g. remove mustCheck).